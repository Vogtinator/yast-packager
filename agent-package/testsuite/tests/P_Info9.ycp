{
return Execute(.setEnvironment,$["packageinfopath":"tests","language":"de","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return Read(.selGroups);
}
{
    return Execute(.selectSelInstall, "Standard", true );
}
{
    return Read(.selInstallSet);
}
{
    return Read(.selDependencies);
}
{
    return Execute(.selectSelInstall, "Minimal", false );
}
{
    return Read(.selDependencies);
}
{
    return Execute(.deselectSelInstall, "Minimal" );
}
{
    return Read(.selDependencies);
}
{
    return Execute(.selectSelInstall, "Kde", false );
}
{
    return Read(.selDependencies);
}
{
    return Read(.selInstallSet);
}
{
    return Execute(.deselectSelInstall, "Games" );
}
{
    return Read(.selDependencies);
}
{
    return Read(.selInstallSet);
}
{
    return Execute(.setInstallSelection,["aaa_base", "aaa_dir", "aaa_skel", "at", "base", "bash", "bash1", "bdflush", "compat", "compress", "cpio", "cracklib", "cron", "ddrescue", "devs", "diff", "eazy", "elvis", "ext2fs", "file", "fileutil", "find", "gawk", "gdbm", "gppshare", "groff", "gzip", "kbd", "kernmod", "kernmods", "ldso", "less", "libaout", "libz", "lilo"], false);
}
{
    return Read(.installSet);
}
{
    return Read(.deleteSet);
}
{
    return Execute(.selectInstall,"loadlin", false);
}
{
    return Execute(.deselectInstall,"lilo");
}
{
    return Read(.installSet);
}
{
    return Read(.selInstallSet);
}
