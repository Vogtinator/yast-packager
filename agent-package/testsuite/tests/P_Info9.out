(true)
([["Devel", " Entwicklung", "addon"], ["Games", " Spiele", "addon"], ["Gnome", " Gnome System", "addon"], ["Kde", " KDE komplett", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Min", " Minimum system", "baseconf"], ["Minimal", " Minimal System", "baseconf"], ["Minimal+X11", " Minimales graphisches System (ohne KDE) ", "baseconf"], ["Multimed", " Multimedia", "addon"], ["Multimedia", " Multimedia", "addon"], ["Network", " Netzwerk/Server", "addon"], ["Standard", " Standard-System", "baseconf"], ["default", " Standard-System mit Office", "baseconf"], ["xxxx", "  german xxxx", "baseconf"], ["zzz_Alles", " Alle Pakete (sehr zeitaufwendig)", "baseconf"]])
(true)
(["Kde-Desktop", "Standard"])
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(true)
($["ADD":[], "CONFLICT":[$["name":"Minimal", "selections":["Standard"]], $["name":"Standard", "selections":["Minimal"]]], "REQUIRE":[]])
(true)
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(true)
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(["Kde", "Kde-Desktop", "Standard"])
(true)
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(["Kde", "Kde-Desktop", "Standard"])
(false)
([["a1/aaa_base.rpm", "SuSE Linux Basispaket", 1, true], ["a1/aaa_dir.rpm", "SuSE Linux Verzeichnisstruktur", 1, true], ["a1/aaa_skel.rpm", "SuSE Linux Default-Benutzer", 1, false], ["a1/base.rpm", "Einige wichtige GNU-Programme", 1, true], ["a1/bash.rpm", "Die GNU Bourne-Again Shell", 1, true], ["a1/bc.rpm", "GNU bc - Kommandozeilen-Rechner", 1, false], ["a1/bdflush.rpm", "Linux-Update-Daemon", 1, true], ["a1/compat.rpm", "Einige Bibliotheken in Kompatibilitätsversionen", 1, false], ["a1/compress.rpm", "Standard Unix-Komprimierungsprogramm", 1, false], ["a1/cpio.rpm", "GNU cpio Backup- und Archiv-Utilities", 1, true], ["a1/cracklib.rpm", "Library, um Passwörter zu checken", 1, true], ["a1/cron.rpm", "Cron-Daemon", 1, true], ["a1/devs.rpm", "Gerätedateien", 1, true], ["a1/file.rpm", "Ermitteln des Dateityps", 1, true], ["a1/gawk.rpm", "GNU gawk", 1, true], ["a1/gdbm.rpm", "GNU Datenbankfunktionen", 1, true], ["a1/glibc.rpm", "Die shared Libraries der C-Library", 1, true], ["a1/gppshare.rpm", "Dynamische C++-Bibliotheken", 1, true], ["a1/groff.rpm", "GNU troff Textformatierungssystem", 1, true], ["a1/gzip.rpm", "GNU zip Komprimierungs-Utilities", 1, true], ["a1/kbd.rpm", "Tastaturtabellen, Consolefonts", 1, true], ["a1/less.rpm", "Programm zum Lesen von Textdateien", 1, false], ["a1/libz.rpm", "Kompressions-Bibliothek", 1, false], ["a1/lilo.rpm", "LILO - Der Bootloader", 1, true], ["a1/ncurses.rpm", "Neue Curses-Bibliotheken", 1, true], ["a1/readline.rpm", "Die readline Bibliothek", 1, true], ["ap1/at.rpm", "Job-Manager", 1, false], ["ap1/gfxboot.rpm", "Grafisches Bootlogo für lilo und syslinux.", 1, false], ["a2/bash1.rpm", "Die GNU Bourne-Again Shell (alte Version)", 2, false], ["a2/eazy.rpm", "Kompatibilitäts-Paket", 2, false], ["a2/termcap.rpm", "Die libtermcap", 2, false], ["ap2/ddrescue.rpm", "Daten kopieren bei Datenfehlern", 2, false], ["ap3/elvis.rpm", "Ein vi-Clone namens elvis", 3, false]])
([])
(true)
(true)
([["a1/aaa_base.rpm", "SuSE Linux Basispaket", 1, true], ["a1/aaa_dir.rpm", "SuSE Linux Verzeichnisstruktur", 1, true], ["a1/aaa_skel.rpm", "SuSE Linux Default-Benutzer", 1, false], ["a1/base.rpm", "Einige wichtige GNU-Programme", 1, true], ["a1/bash.rpm", "Die GNU Bourne-Again Shell", 1, true], ["a1/bc.rpm", "GNU bc - Kommandozeilen-Rechner", 1, false], ["a1/bdflush.rpm", "Linux-Update-Daemon", 1, true], ["a1/compat.rpm", "Einige Bibliotheken in Kompatibilitätsversionen", 1, false], ["a1/compress.rpm", "Standard Unix-Komprimierungsprogramm", 1, false], ["a1/cpio.rpm", "GNU cpio Backup- und Archiv-Utilities", 1, true], ["a1/cracklib.rpm", "Library, um Passwörter zu checken", 1, true], ["a1/cron.rpm", "Cron-Daemon", 1, true], ["a1/devs.rpm", "Gerätedateien", 1, true], ["a1/file.rpm", "Ermitteln des Dateityps", 1, true], ["a1/gawk.rpm", "GNU gawk", 1, true], ["a1/gdbm.rpm", "GNU Datenbankfunktionen", 1, true], ["a1/glibc.rpm", "Die shared Libraries der C-Library", 1, true], ["a1/gppshare.rpm", "Dynamische C++-Bibliotheken", 1, true], ["a1/groff.rpm", "GNU troff Textformatierungssystem", 1, true], ["a1/gzip.rpm", "GNU zip Komprimierungs-Utilities", 1, true], ["a1/kbd.rpm", "Tastaturtabellen, Consolefonts", 1, true], ["a1/less.rpm", "Programm zum Lesen von Textdateien", 1, false], ["a1/libz.rpm", "Kompressions-Bibliothek", 1, false], ["a1/ncurses.rpm", "Neue Curses-Bibliotheken", 1, true], ["a1/readline.rpm", "Die readline Bibliothek", 1, true], ["ap1/at.rpm", "Job-Manager", 1, false], ["a2/bash1.rpm", "Die GNU Bourne-Again Shell (alte Version)", 2, false], ["a2/eazy.rpm", "Kompatibilitäts-Paket", 2, false], ["a2/loadlin.rpm", "Linux Loader mit Kommandozeile von DOS", 2, false], ["a2/termcap.rpm", "Die libtermcap", 2, false], ["ap2/ddrescue.rpm", "Daten kopieren bei Datenfehlern", 2, false], ["ap3/elvis.rpm", "Ein vi-Clone namens elvis", 3, false]])
(["Kde", "Kde-Desktop", "Standard"])
